modules/st2-auto-form/auto-form.component.js

```diff
@@ -54,7 +54,7 @@ export default class AutoForm extends React.Component {
 
   handleChange(name, value) {
     const { spec, data, onChange } = this.props;
-    return onChange({
+    return onChange && onChange({
```

Comment:
While this removes the error, it masks the underlying issue. As this is a controlled component, it must have an onChange value.

Unfortunately, it can also be used without onChange if disabled is set to true, which is why it doesn't have isRequired. I have created a [PR](https://github.com/StackStorm/st2web/pull/463) that adjusts the propTypes accordingly.